modules/client/src/allowlistVoting/context.ts

```diff
@@ -0,0 +1,20 @@
+import { ContextPlugin, ContextPluginParams } from "../client-common";
+import {
+  IAllowlistVotingContextParams,
+  IAllowlistVotingContextPluginState,
+} from "./interfaces";
+
+export class AllowlistVotingContextPlugin extends ContextPlugin<
+  IAllowlistVotingContextPluginState
+> {
+  constructor(
+    params: Partial<ContextPluginParams> & IAllowlistVotingContextParams
+  ) {
+    super(params);
+    this.state.pluginAddress = params.pluginAddress;
+  }
+
+  get pluginAddress(): string {
+    return this.state.pluginAddress;
```

Comment:
AFAIK we agreed to pass the plugin address as a parameter to the functions, following the pattern in the client

Reply:
done